lib/api/core/plugins/pluginContext.js

```diff
@@ -148,6 +148,14 @@ class PluginContext {
         get: () => curryTrigger(kuzzle, pluginName)
       });
 
+      const throwNotAvailable = name => () => {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "throwNotAvailable" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'throwNotAvailable' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AUnusedVariable)